src/api/status/public/js/build-log/terminal.js

```diff
@@ -66,4 +65,28 @@ const copyToClipboard = async () => {
 };
 copyToClipboardButton.addEventListener('click', () => copyToClipboard());
 
+/**
+ * Resize the terminal depending on the height of the window
+ */
+const resizeTerminal = (function () {
```

Comment:
Can we do this without using `currying` (not sure if it's the correct name for this pattern). It introduces unnecessary complexity

Reply:
I think you meant using a closure. I am going to try with the CSS approach mentioned by @DukeManh, and if it does not work, I will rewrite this to be a little bit simpler.